### ISSUE TITLE
Bug - Webpack - Sass Compile - Unknown word $themeColor

### ISSUE THREAD

**Issue by samuelecarpene** [nolimits4web/swiper#3317]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 5.1.0 (https://registry.npmjs.org/swiper/-/swiper-5.1.0.tgz).
* Platform/Target and Browser Versions: Ubuntu 18 Lamp + Node v10.16.3 + Webpack 4.41.2.

### What i did
I installed swiper via npm.
Included swiper scss in my main.scss (main scss file of site).
Configured webpack to compile sass/scss to css.

File webpack.config.js

```js
const MiniCssExtractPlugin = require('mini-css-extract-plugin');

...


let webpackConfig = {
...

 module: {
        rules: [
             {
                test: /\.(sa|sc|c)ss$/,
                use: [
                    {
                        loader: MiniCssExtractPlugin.loader,
                        options: {
                            hmr: process.env.NODE_ENV === 'development',
                        },
                    },
                    // 'css-loader',
                    // 'resolve-url-loader',
                    //'postcss-loader',
                    //'sass-loader',
                    {
                        loader: 'css-loader',
                        options: {
                            sourceMap: true
                        }
                    },
                    {
                        loader: 'postcss-loader',
                        options: {
                            sourceMap: true
                        }
                    },
                    {
                        loader: 'sass-loader',
                        options: {
                            sourceMap: true
                        }
                    }
                ],
            }
,
```

File postcss.config.js

```js
module.exports = {
    plugins: {
        'postcss-custom-properties': { },
        'autoprefixer': {}
    }
}
```

### Expected Behavior
Compile scss to css

### Actual Behavior
Compilation throw an error "Unknown word $themeColor" caused by trying to compile this block

```scss
:root {
  --swiper-theme-color: $themeColor;
}
```

I checked bootstrap 4 rules 
https://github.com/twbs/bootstrap/blob/master/scss/_root.scss
that are compiled correctly

Maybe the problem can be related in the fact that the var $themeColor; is not interpolated (https://sass-lang.com/documentation/interpolation)

I made a test using interpolation
```scss
:root {
  --swiper-theme-color: #{$themeColor};
}
```
and all is fine.

The same problem appear in a project that use laravel mix (https://github.com/JeffreyWay/laravel-mix) including swiper.scss in main project scss.

Thank you in advance.

**Comment by gfnool** [comment]
Same problem but with a different output: I'm using gulp-sass and the $themeColor variable is not resolved.

The interpolation is also suggested in the sass documentation:
https://sass-lang.com/documentation/breaking-changes/css-vars

Thank you

**Comment by nolimits4web** [comment]
Fixed by #3334
